src/Tribe/Main.php

```diff
@@ -121,6 +121,9 @@ class_exists( 'TribeEvents', false )
 		$this->rsvp();
 
 		$this->user_event_confirmation_list_shortcode();
+
+		// Load the Hooks on JSON_LD
+		Tribe__Tickets__JSON_LD__Order::hook();
```

Comment:
Isn't `hook()` an instance method? We should either change how we call it or change the method signature.